prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+    <div id="h1">Hello World!</div>
```

Comment:
For this exercise we don't want to modify the HTML itself, but instead use JavaScript code to add a new element to the DOM. I recommend completing the dom manipulation and dom events exercises first, as our exercises are meant to be completed from bottom to top in your dashboard.

Reply:
No wonder I was having trouble. I'll do that today.